src/button/example/index.ts

```diff
@@ -4,9 +4,19 @@ import { StatefulMixin } from '@dojo/widget-core/mixins/Stateful';
 import { ProjectorMixin } from '@dojo/widget-core/mixins/Projector';
 import { w, v } from '@dojo/widget-core/d';
 import Button, { ButtonType } from '../../button/Button';
+import dojo from '../../themes/dojo/theme';
```

Comment:
This import may be better named something like `dojoTheme` to avoid confusion. Applies to all example pages.